gamemode/modules/doorsystem/sh_doors.lua

```diff
@@ -1,28 +1,36 @@
 local meta = FindMetaTable("Entity")
 local plyMeta = FindMetaTable("Player")
 
+local doorsOwnable = {
```

Comment:
Perhaps a minor technical detail. This code is easier to read, but might not necessarily be faster. Table lookups with string involve some hashing function. Such hashing function might be ever slightly more expensive for small tables than a linear if statement checking all entries.

Regardless, this change is good, as it improves readability, and is pretty much as fast as the other implementation. Good work 👍

Reply:
If I may ask, why are there unownable doors which cannot really be interacted with?

Reply:
The answer to that question lies many years back. I believe there were maps with those entity types which for some reason should be treated as doors. I don't know which maps anymore, and tbh I don't know what, if anything, will break if this is changed.